proto/viam/app/v1/billing.proto

```diff
@@ -28,26 +28,6 @@ message InvoiceSummary {
   google.protobuf.Timestamp paid_date = 6;
 }
 
-message BillableResourceEvent {
```

Comment:
These should still be removed as they are not used anywhere and have never been so it doesn't make sense to even "deprecate" them